test/langtools/tools/javac/processing/ReportOnImportedModuleAnnotation/ReportOnImportedModuleAnnotation.java

```diff
@@ -49,6 +58,25 @@ public static void main(String[] args) throws Exception {
 
         final JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
 
+        // Clean any existing class files in output directory
+        var tb = new ToolBox();
+        Files.walkFileTree(testOutputPath,
+                           new SimpleFileVisitor<Path>() {
+                               @Override
+                               public FileVisitResult visitFile(Path path,
+                                                         BasicFileAttributes attrs) {
+                                   File file = path.toFile();
+                                   if (file.getName().endsWith(".class")) {
```

Comment:
You can avoid the `File` with:

    if (path.getFileName().toString().endsWith(".class"))